Packs/SilentPush/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Silent Push",
     "description": "The Silent Push platform focuses on proactive threat intelligence and threat hunting.",
     "support": "partner",
-    "currentVersion": "1.1.0",
+    "currentVersion": "2.0.0",
```

Comment:
it's not major change in the pack, a minor update should be good, or even a revision, so please change it to 1.2.0 or 1.1.1